app/views/shared/_display_ad_dropdown_content.html.erb

```diff
@@ -0,0 +1,14 @@
+<ul class="list-none m-0">
+    <li>
```

Comment:
This is doing exactly what the issue requests, and yet, with this link as a 404, we probably shouldn't deploy this. What about adding something so that it becomes valid as soon as `/billboard` exists? Something like:

```suggestion
        <% if Page.exists?(slug: "billboard") %>
```

We'd have a dropdown with only one item, but I think that'd be better than a link that goes nowhere. What do you think?

Reply:
Done. I have added this check and also verified that it does work.
<img width="362" alt="Screenshot 2023-02-20 at 2 39 44 PM" src="https://user-images.githubusercontent.com/9396084/220081934-8e40d4e4-ba35-43a8-b79e-3b767df119d4.png">